packages/components/data-table-manager/src/data-table-manager.tsx

```diff
@@ -181,9 +233,19 @@ const DataTableManager = <Row extends TRow = TRow>(
           areDisplaySettingsEnabled && props.displaySettings!.isCondensed,
       })}
     </Spacings.Stack>
+  ) : (
+    <DataTableSettings
+      topBar={props.topBar}
+      onSettingsChange={props.onSettingsChange}
+      columnManager={props.columnManager}
+      displaySettings={props.displaySettings}
+      managerTheme="light"
+    />
```

Comment:
We can just keep the previous code and conditionally render the `children`.

Reply:
Updated here: https://github.com/commercetools/ui-kit/pull/2807/commits/bd8447871f0d99ac366b5adc9d865055c9a4506e

Thank you